packages/aws-cdk-lib/aws-s3/lib/bucket.ts

```diff
@@ -2190,13 +2193,24 @@ export class Bucket extends BucketBase {
     }));
   }
 
-  private parseOwnershipControls({ objectOwnership }: BucketProps): CfnBucket.OwnershipControlsProperty | undefined {
-    if (!objectOwnership) {
+  private parseOwnershipControls(
+    objectOwnership?: ObjectOwnership,
+    accessControl?: BucketAccessControl,
+  ): CfnBucket.OwnershipControlsProperty | undefined {
```

Comment:
For style, since this function is only ever called with `this.whatever`, it doesn't need to take these as parameters.

It could just read them from the object immediately.